src/erlang17/plan.sh

```diff
@@ -0,0 +1,54 @@
+pkg_name=erlang17
+pkg_origin=core
+pkg_version=17.5
+pkg_description="A programming language for massively scalable soft real-time systems."
+pkg_upstream_url="http://www.erlang.org/"
+pkg_dirname=otp_src_${pkg_version}
+pkg_license=('Apache-2.0')
+pkg_maintainer="Elliott Davis <edavis@chef.io>"
+pkg_source=http://www.erlang.org/download/otp_src_${pkg_version}.tar.gz
+pkg_filename=otp_src_${pkg_version}.tar.gz
+pkg_shasum=3c28820c981b30c50df8ac2a4a238e5813454fa3ed2ad530bc7948a282846474
+pkg_deps=(core/glibc core/zlib core/ncurses core/openssl core/sed)
+pkg_build_deps=(core/coreutils core/gcc core/make core/openssl core/perl core/m4)
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+
+do_prepare() {
+  # The `/bin/pwd` path is hardcoded, so we'll add a symlink if needed.
+  if [[ ! -r /bin/pwd ]]; then
+    ln -sv "$(pkg_path_for coreutils)/bin/pwd" /bin/pwd
+    _clean_pwd=true
+  fi
+
+  if [[ ! -r /bin/rm ]]; then
+    ln -sv "$(pkg_path_for coreutils)/bin/rm" /bin/rm
+    _clean_rm=true
+  fi
+}
+
+do_build() {
+  ./configure --prefix="${pkg_prefix}" \
+              --enable-threads \
+              --enable-smp-support \
+              --enable-kernel-poll \
+              --enable-dynamic-ssl-lib \
+              --enable-shared-zlib \
+              --enable-hipe \
+              --with-ssl="$(pkg_path_for openssl)/lib" \
+              --with-ssl-include="$(pkg_path_for openssl)/include" \
+              --without-javac
```

Comment:
It isn't vital but we can save a bit of space by not including `wc` and `megaco`: https://github.com/chef/omnibus-software/blob/master/config/software/erlang.rb#L137-L139